### DISCUSSION TITLE
Value to custom field from custom sccript

### DISCUSSION THREAD

**Discussion by nyevess** [netbox-community/netbox#13317] [General]
Hello Community,
I'm triying to add a value in a custom script for a custom field, but I don't now the exact syntaxis required, could you help me?

![image](https://github.com/netbox-community/netbox/assets/69692460/79ddd9fe-cde5-4766-b733-240ae58b5894)

**Comment by jcollie** [comment]
custom_fields is a dictionary, so you'd write `custom_fields={"Services": service}`

**Reply by candlerb** [reply]
I think it's `custom_field_data`.

Note that you should do `new_vm.full_clean()` before `new_vm.save()`. It's a major flaw of the Django ORM (IMO) that it doesn't validate models on save.  If you forget this step, you can end up bypassing all model validation and storing junk in the database.
